pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -317,23 +317,29 @@ in (buildEnv {
   ''
     rm "$out"/bin/*-sys
     wrapBin
-  '' +
+  ''
+  + bin.cleanBrokenLinks +
```

Comment:
(Incidental: `cleanBrokenLinks` will disappear with the bin container patch. I believe it is already a no-op since we started using `texlinks`.)